### PR TITLE
logind: introduce CanIdle/CanLock properties on logind session dbus objects

### PR DESCRIPTION
**Pull request by poettering**
Fixes: #34844

### CODE REVIEW

File: man/org.freedesktop.login1.xml [hunk 1]
```diff
@@ -1538,6 +1546,9 @@ node /org/freedesktop/login1/session/1 {
 
       <para><varname>LockedHint</varname> shows the locked hint state of this session, as set by the
       <function>SetLockedHint()</function> method described above.</para>
+
+      <para><varname>CanIdle</varname> indicates whether the session supports the idle hint concept. Similar,
```
Comment (YHNdnzj) [inline man/org.freedesktop.login1.xml]: Similarly?

**Review by YHNdnzj — APPROVED** [review]

File: man/org.freedesktop.login1.xml [hunk 2]
```diff
@@ -1619,10 +1631,9 @@ node /org/freedesktop/login1/session/1 {
       <function>CanSleep()</function>,
       <varname>SleepOperation</varname>, and
       <function>ListSessionsEx()</function> were added in version 256.</para>
-      <para><varname>HandleSecureAttentionKey</varname>,
-      <function>SecureAttentionKey()</function>,
-      <varname>PreparingForShutdownWithMetadata</varname>, and
-      <varname>DesignatedMaintenanceTime</varname> were added in version 257.</para>
+      <para><varname>HandleSecureAttentionKey</varname>, <function>SecureAttentionKey()</function>,
+      <varname>PreparingForShutdownWithMetadata</varname>, <varname>DesignatedMaintenanceTime</varname>,
+      <varname>CanIdle</varname> and <varname>CanLock</varname> were added in version 257.</para>
```
Comment (yuwata) [inline man/org.freedesktop.login1.xml:1636]: I'd like to insert a comma before `and`.

**Review by yuwata — COMMENTED** [review]
LGTM.

File: man/org.freedesktop.login1.xml (hunk 2 above)
Comment (yuwata) [reply man/org.freedesktop.login1.xml:1636]: -> e7c567cc78b7e7e114ee5d86e717623fc006f067

**Pull request merged**